Expand on debugging methods in README (#4229)
realm/realm-js: README.md

## Known issues

> * Realm is not compatible with the Chrome Debugger.  The following debugging methods are supported.

Reviewer: ```suggestion
* Support for the Chrome Debugger has been deprecated.  The following debugging methods are supported.
```
